tests/components/nx584/test_binary_sensor.py

```diff
@@ -54,17 +61,16 @@ def test_setup_defaults(self, mock_nx, mock_watcher):
             "exclude_zones": [],
             "zone_types": {},
         }
-        assert nx584.setup_platform(self.hass, config, add_entities)
-        mock_nx.assert_has_calls(
-            [mock.call(zone, "opening") for zone in self.fake_zones]
-        )
+        assert nx584.setup_platform(hass, config, add_entities)
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)